tensorrt_llm/_torch/auto_deploy/models/custom/modeling_nemotron_flash.py

```
@@ -0,0 +1,1090 @@
+                )
+            else:
+                deltanet_idx = layer_idx
+
+            self.gla = DeltaNet(
+                hidden_size=config.hidden_size,
+                num_heads=config.num_attention_heads,
+                layer_idx=deltanet_idx,
+                config=self.config,
+            )
+        else:
+            raise ValueError(f"Not supported: {config.hybrid_decoder_layer}")
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)